### PR TITLE
Improve roadmap page visually

### PR DESCRIPTION
**Pull request by GeckoEidechse**
Defines roadmap items in JSON file and loads them on page load.

Shows each roadmap item as a card with a banner image, title, and description.

Rendered:

image

### CODE REVIEW

File: dist/script/roadmap.js [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+        <div class="roadmap-card-title">TITLE</div>
+        <div class="roadmap-card-text">DESCRIPTION</div>
+    </div>
+</div>
+`
+
+function addRoadmapItem(group, icon, image_alt_text, title, description) {
+
+    var x = template.replace("IMAGE", icon)
+        .replace("IMAG_ALT_TEXT", image_alt_text)
```
Comment (barnabwhy) [inline dist/script/roadmap.js:14]: Typo here, alt text won't actually get set

File: dist/style/roadmap.css [hunk 2]
```diff
@@ -0,0 +1,34 @@
+.roadmap-container {
+    display: flex;
+    flex-wrap: wrap;
+    gap: 20px;
+    justify-content: center;
+}
+
+.roadmap-card {
+    background-color: white;
```
Comment (barnabwhy) [inline dist/style/roadmap.css:9]: The white background looks kind of out of place to me, perhaps going for dark blue with white text is better?

Comment (GeckoEidechse) [reply dist/style/roadmap.css:9]: Will give it a try o7

Although I guess at this point this PR has been superseded by #75 lol

**Comment by EM4Volts** [comment]
Like the overall Page. 
jut have the small idea of maybe also communicating wether a feature is activly being worked on and maybe also when its currently aprox. going to be done.

i made a small mockup on how the new capsules could look for it, in my idea the tags are optional. so if no date is preset or agreed on yet then the tag wouldnt be there, same as the indev tag
Screenshot 2024-07-10 005531

**Comment by GeckoEidechse** [comment]
> i made a small mockup on how the new capsules could look for it \[...\]

Oooooh I like that lot. Not sure if we can should give a date as we often don't know ourselves but an approximation might be enough ig ^^"

Was thinking of also somehow integrating a way of saying who's working on that feature and pushing it forward (e.g. @Zanieon for FD atm)

**Comment by Zanieon** [comment]
I totally agree on pointing out which developer is doing what, at least whos focused in that specific department.

**Comment by GeckoEidechse** [comment]
Superseded by #75 and suggestions from this PR not yet implemented moved to #77

**Pull request closed without merging**